scripts/get-condition-vars.js

```diff
@@ -1,15 +1,16 @@
 /* eslint-disable import/no-commonjs */
 
 module.exports = core => {
-  const { CLIENT_PAYLOAD_ARG, ENABLE_CACHE_ARG } = process.env
+  const { IS_NON_COMMIT_ARG, ENABLE_CACHE_ARG } = process.env
   try {
-    const payload = JSON.parse(CLIENT_PAYLOAD_ARG)
-    const isNonCommitEvent = payload.isNonCommitEvent === true
-    const skipGitClone = isNonCommitEvent && ENABLE_CACHE_ARG === 'true'
-    core.exportVariable('IS_NON_COMMIT_EVENT', isNonCommitEvent.toString())
+    const skipGitClone =
```

Comment:
are you sure? looks like you comparing `bool` to `string`.

Reply:
yeah, I debugged it, and it turns out that envs are being passed as strings